.travis.yml

```diff
@@ -23,6 +23,6 @@ before_script:
 
 script:
 # Unable to initialize database, use the following code to debug. - docker-compose up --build
-#- docker-compose up --build
+- docker-compose up --build
```

Comment:
change this line "- docker-compose up --build"
to "#- docker-compose up --build"